x/dex/keeper/abci/end_block_deposit.go

```diff
@@ -52,7 +52,7 @@ func (w KeeperWrapper) GetDepositSudoMsg(ctx sdk.Context, typedContractAddr type
 	if err != nil {
 		panic(err)
 	}
-	if err := w.BankKeeper.DeferredSendCoinsFromModuleToAccount(ctx, types.ModuleName, contractAddr, escrowed); err != nil {
+	if err := w.BankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, contractAddr, escrowed); err != nil {
```

Comment:
this is okay since deposit is handled sequentially

Reply:
I think we updated it to be concurrent with the overall dex endblock optimization

Reply:
https://github.com/sei-protocol/sei-chain/blob/52cf5679e95f9e491596b62588a8525b4bcd1647/x/dex/contract/abci.go#L179-L194